controller_manager:
  ros__parameters:
    update_rate: 100

    arm_position_controller:
      type: joint_trajectory_controller/JointTrajectoryController
      joints:
        - arm_link_joint
      interface_name: position

arm_position_controller:
  ros__parameters:
    joints:
      - arm_link_joint
    interface_name: position
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
    gains:
      arm_link_joint:
        p: 20.0
        i: 5.0
        d: 1.0
    constraints:
      goal_time: 0.5
      stopped_velocity_tolerance: 0.01
      arm_link_joint:
        trajectory: 0.1
        goal: 0.1
